Add automated tests with Molecule

ajdecon opened the pull request:
Adds some basic automated tests with [Ansible Molecule](https://molecule.readthedocs.io/en/latest/index.html), as well as a Github action to execute them on push and PR.

Also makes some minor changes to the role to get the tests passing. 😉

ajdecon committed:
Add basic molecule tests

ajdecon committed:
remove centos-8 tests for now

ajdecon committed:
add github action

lukeyeager reviewed (approved):
Looks slick! https://github.com/ajdecon/ansible-role-enroot/actions/runs/1655877404

lukeyeager commented on molecule/default/molecule.yml:
```diff
@@ -0,0 +1,19 @@
+    pre_build_image: true
+  - name: centos-7
+    image: geerlingguy/docker-centos7-ansible
+    pre_build_image: true
```
Could do centos8 too, right? It's listed in `meta/main.yml`, at least.

ajdecon committed:
Add CentOS-8 to list of platforms for molecule

Also minor tweaks to the install process, as the container for CentOS 8
is pickier about GPG key validation than CentOS 7.

Note that we disable the GPG check when installing Enroot because the
packages on Github are not signed.

ajdecon commented:
Added CentOS 8! And confirmed that all tests pass on this branch. 🎉 https://github.com/ajdecon/ansible-role-enroot/runs/4717679877

kftsehk commented on tasks/redhat.yml:
```diff
@@ -41,29 +46,36 @@
     yum:
       name: "{{ item }}"
       state: "{{ enroot_package_state }}"
+      disable_gpg_check: true
```
fixes #19 #17
